spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkTable.java

```diff
@@ -289,14 +288,10 @@ public void deleteWhere(Filter[] filters) {
       return;
     }
 
-    try {
-      icebergTable.newDelete()
-          .set("spark.app.id", sparkSession().sparkContext().applicationId())
-          .deleteFromRowFilter(deleteExpr)
-          .commit();
-    } catch (ValidationException e) {
-      throw new IllegalArgumentException("Failed to cleanly delete data files matching: " + deleteExpr, e);
-    }
+    icebergTable.newDelete()
+            .set("spark.app.id", sparkSession().sparkContext().applicationId())
```

Comment:
Can we keep the same indent level (4 spaces)?

Reply:
Added the code style XML to my IDE, it's fixed now.